src/simsopt/mhd/vmec_diagnostics.py

```diff
@@ -986,18 +994,147 @@ def residual(theta_v, phi0, theta_p_target, jradius):
         return theta_p_target - theta_p
         """
         return theta_p_target - (theta_v + np.sum(lmns[jradius, :] * np.sin(xm * theta_v - xn * phi0)))
-
-    # Solve for theta_vmec corresponding to theta_pest:
+    
+    # A helper function that returns 0 when theta_vmec_trys are close to the
+    # proper toroidal angle in magnetic coordinates
```

Comment:
I don't think this description is accurate. The function returns 0 when `theta_vmec_trys` correspond to points along a field line, right?